.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+
+# One line description
```

Comment:
I think the PR title already functions as the "One line description".

I'd suggest changing this heading to "``# Description``" or "``#Brief description``"

Reply:
You're right, the PR title is the one-line description.

Reply:
did it